src/cassio/table/mixins/clustered.py

```diff
@@ -87,19 +94,19 @@ def _normalize_kwargs(self, args_dict: Dict[str, Any]) -> Dict[str, Any]:
             if value is None:
                 continue
 
-            if not isinstance(value, tuple):
-                value = (value,)
-
-            if len(value) == 1:
-                new_args_dict[col] = value[0]
-            else:
+            if isinstance(value, tuple):
                 for i, v in enumerate(value):
                     new_args_dict[f"{col}_{i}"] = v
+            else:
+                new_args_dict[col] = value
```

Comment:
will this work if value is a tuple of length one?

Reply:
The logic here has been changed to avoid these kinds of ambiguities.
In general the only case for a len-1 tuple should be specifying only the first value of a multi-col clustering key for a get_partition. There, the caller _must_ pass `row_id=(value,)` for the unpacking to work.